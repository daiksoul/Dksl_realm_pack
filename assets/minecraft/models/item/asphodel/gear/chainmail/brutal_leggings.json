{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/chainmail_leggings",
		"layer1": "misc/brutal_symbol"
	}
}
