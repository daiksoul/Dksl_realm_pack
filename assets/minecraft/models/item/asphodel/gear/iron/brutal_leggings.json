{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/iron_leggings",
		"layer1": "misc/brutal_symbol"
	}
}
