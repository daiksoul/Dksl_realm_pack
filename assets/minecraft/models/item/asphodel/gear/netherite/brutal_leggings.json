{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/netherite_leggings",
		"layer1": "misc/brutal_symbol"
	}
}
