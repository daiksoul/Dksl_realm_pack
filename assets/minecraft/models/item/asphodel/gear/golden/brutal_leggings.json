{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/golden_leggings",
		"layer1": "misc/brutal_symbol"
	}
}
